lbz/response.py

```diff
@@ -17,21 +21,37 @@ def __init__(
         base64_encoded: bool = False,
     ):
         self.body = body
```

Comment:
Also, it can be simplified:
```suggestion
        self.body = body
        self._json = body if isinstance(body, dict) else None
```